core/src/main/kotlin/pages/contentNodeProperties.kt

```diff
@@ -7,10 +7,40 @@ class SimpleAttr(val extraKey: String, val extraValue: String) : ExtraProperty<C
     override val key: ExtraProperty.Key<ContentNode, SimpleAttr> = SimpleAttrKey(extraKey)
 
     companion object {
-        fun header(value: String) = SimpleAttr("data-togglable", value)
+        fun togglableTarget(value: String) = SimpleAttr(TOGGLEABLE_CONTENT_TYPE_ATTR, value)
     }
 }
 
+object HtmlInvisibleExtra : ExtraProperty<ContentNode>, ExtraProperty.Key<ContentNode, HtmlInvisibleExtra> {
+    override val key: ExtraProperty.Key<ContentNode, *> = this
+}
+
+const val TOGGLEABLE_CONTENT_TYPE_ATTR = "data-togglable"
+enum class BasicToggleableContentType : ToggleableContentType {
+    TYPE, CONSTRUCTOR, FUNCTION, PROPERTY, ENTRY, EXTENSION, INHERITED_FUNCTION, INHERITED_PROPERTY
+}
+
+interface ToggleableContentType
```

Comment:
> It now groups some content block

For this purpose, we have `DCI.kind`.
I do want to mix different types of classification of content, so I do not use `DCI.kind` for tabs.

> It now groups some content block,

Currently, it is enough. 

> It also looks a bit inconsistent to have INHERITED_FUNCTION, but no EXTENSION_FUNCTION.
> I would propose to mark blocks with two types - the main one (function/property) and an additional (extension/inherited), this should give more flexibility.

And `INHERITED_EXTENSION_FUNCTION`.  It adds extra complexity when it does not need for us. Let's follow Occam's razor.
In my opinion, we should not have `INHERITED_FUNCTION, INHERITED_PROPERTY` here, but I left it due to having a configuration flag.

Reply:
Oh yeah, I forgot about the other `ContentKind`... Still proposing to rename it to be a bit more universal - `TabbedContentType`. 

The `toggleable` abstraction, if I'm not mistaken, exists only in the HTML layer (because of the style name). In the Markdown format it would make a little less sense (new abstraction./name) and it won't be consistent with `ContentTab`.

Reply:
`ToggleableContentType` is more universal. I want to use it, e.g. for checkboxes as well.
 In the Markdown format `tabbed` name has no sense too.

Reply:
I meant universal as in "universal across formats", not "has more than one usage". It is a bit confusing to see that `ContentTab` has `ToggleableContentType`, as the difference between a "tab" and a "toggleable" is not that clear. For me it'd cause less questions if it was narrowed down (i.e `TabbedContetType`) or very wide (i.e `ContentKind`, but that exists already). Otherwise I feel like we need to document the difference/distinction such that the API is not misused